src/components/Radio/index.tsx

```diff
@@ -0,0 +1,107 @@
+import type { ChangeEvent, HTMLAttributes, ReactElement } from 'react'
+import styled from '@emotion/styled'
+import type { StyledProps } from '@types'
+
+export interface RadioProps extends HTMLAttributes<HTMLFormElement> {
+  formName: string
+  items: { code: string; name: string }[]
+  radioDirection: 'horizontal' | 'vertical'
+  onChangeRadioButton(e: ChangeEvent<HTMLInputElement>): void
+}
```

Comment:
```suggestion
export interface RadioProps extends HTMLAttributes<HTMLFormElement> {
  formName: string
  items: { code: string; name: string }[]
  direction: 'horizontal' | 'vertical'
  onChange(e: ChangeEvent<HTMLInputElement>): void
}
```
radio로 전달하는 요소들이기 때문에 축약해도 좋을 것 같습니다!

Reply:
오 좋습니닷!! 그런데 onChange같은경우에 input의 onChange라서 form의 prop으로 넘길때
HTMLFormElement 타입이 거부를 해서요 ㅠㅠ 변수명 변경이 불가피 할 것 같습니닷 ㅠㅠ

Reply:
@shinhyojeong 요기도!